libraries/pod/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 pub mod bytemuck;
 pub mod error;
 pub mod optional_keys;
+pub mod pod_option;
```

Comment:
nit: can this file be called `option`? otherwise `spl_pod::pod_option::PodOption` has the word `Pod` a few too many times :sweat_smile: